packages/nuxt/src/plugins/plugin.ts

```diff
@@ -16,12 +16,29 @@ import { loadConfigFile } from '@nx/devkit/src/utils/config-utils';
 import { getNamedInputs } from '@nx/devkit/src/utils/get-named-inputs';
 import { calculateHashForCreateNodes } from '@nx/devkit/src/utils/calculate-hash-for-create-nodes';
 import { workspaceDataDirectory } from 'nx/src/utils/cache-directory';
+import { normalizeOptions } from 'nx/src/utils/normalize-options';
 import { getLockFileName } from '@nx/js';
 import { dirname, isAbsolute, join, relative } from 'path';
 import { existsSync, readdirSync } from 'fs';
 import { loadNuxtKitDynamicImport } from '../utils/executor-utils';
 import { addBuildAndWatchDepsTargets } from '@nx/js/src/plugins/typescript/util';
 
+export interface NuxtPluginOptions {
+  buildTargetName?: string;
+  serveTargetName?: string;
+  serveStaticTargetName?: string;
+  buildStaticTargetName?: string;
+  buildDepsTargetName?: string;
+  watchDepsTargetName?: string;
```

Comment:
buildDepsTargetName
watchDepsTargetName

are missing from the defaults, see also webpack options behavior

Reply:
they got added in this commit but only webpack got an update to the defaultOptions:

https://github.com/nrwl/nx/commit/0ae8665a885935982ecbfc3683b3d38bc4d56f44#diff-8ea87d5fdfbfcec0fc69136da5a959e64f7e2747a01611c16227183d06b24eabR33-R331